src/components/DateMonthPicker/index.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react'
+import { Button, translate, Icon, Bold } from 'cozy-ui/transpiled/react'
+import range from 'lodash/range'
+import { format } from 'date-fns'
+
+const gridStyle = {
+  display: 'grid',
+  gridTemplateColumns: 'repeat(4, auto)',
+  gridTemplateRows: 'repeat(3, 3fr)',
+  overflow: 'hidden'
+}
+
+const yearButtonStyle = {
+  flexGrow: 0,
+  minWidth: '3rem',
+  minHeight: '3rem',
+  textAlign: 'center',
+  borderWidth: 0,
+  margin: 0,
+  color: 'var(--coolGrey)',
+  background: 'none !important'
```

Comment:
`!important` is needed even with inline styles?

Reply:
It was for the hover. In cozy-ui I added a rule for the hover. No !important :)